File: app/models/recipe.py
from sqlalchemy import Column, Integer, String, Text
from sqlalchemy.types import JSON
from app.database import Base

class Recipe(Base):
    __tablename__ = "recipes"
    id = Column(Integer, primary_key=True, index=True)
    title = Column(String, nullable=False)
    description = Column(String, nullable=False)
    category = Column(String, nullable=False)
    prep_time = Column(Integer)
    cook_time = Column(Integer)
    servings = Column(Integer)
    ingredients = Column(Text)
    instructions = Column(Text)

File: models/Recipe.js
class Recipe {
  constructor({ id, name, category, ingredients, instructions, prepTime }) {
    this.id = id;
    this.name = name;
    this.category = category;
    this.ingredients = ingredients;
    this.instructions = instructions;
    this.prepTime = prepTime;
  }
}

export default Recipe;